core/unix/injector.c

```diff
@@ -862,7 +862,7 @@ static const enum_name_pair_t pt_req_map[] = { { PTRACE_TRACEME, "PTRACE_TRACEME
 #    endif
                                                { PTRACE_ATTACH, "PTRACE_ATTACH" },
                                                { PTRACE_DETACH, "PTRACE_DETACH" },
-#    ifndef AARCH64
+#    if defined(PTRACE_GETFPXREGS) && defined(PTRACE_SETFPXREGS)
```

Comment:
(I thought I sent this out a long time ago but it was sitting pending)

Reply:
> nit: It seems cleaner to have a separate define check for each of these, rather than bundling them, right?

I thought about it. But, it would be really strange to me if one of them was defined but not the other. I assume that both need to come together, so bundling them is meant to reduce verbosity.

Nevertheless, if you find it more clear to have them separated, I can update it ASAP.

Reply:
OK either way.  Will just merge it as-is.